src/BloomExe/Publish/PdfViewer.cs

```diff
@@ -24,17 +24,26 @@ public PdfViewer()
 			InitializeComponent();
 
 #if !__MonoCS__
-			if (Settings.Default.UseAdobePdfViewer)
-			{
-				_pdfViewerControl = new AdobeReaderControl();
-			}
-			else
+			// In Windows we would prefer to use Acrobat to display and print PDFs. It avoids various bugs in
+			// PDFjs, such as BL-1177 (Andika sometimes lost when printing directly from Bloom),
+			// BL-1170 Printing stops after certain point
+			// BL-1037 PDFjs sometimes fails to display if use certain jpg images
+			// If Acrobat is not installed, it will fall back to PDFjs, and we hope for the best.
+			// Todo: we need a better solution in Linux, also. Ghostscript might provide something but
```

Comment:
Does AdobeReaderControl() itself fall back to using GeckoWebBrowser() internally?